DEVELOPERS.md

```diff
@@ -73,3 +73,53 @@ Starting with version 4.0, Bash is licenced under GPLv3. Because of this, macOS
 ```bash
 brew install bash
 ```
+
+
+## Running tests against Databricks
+
+The test suite for Databricks/Spark backend by default runs tests against
+a local spark db in a container, for reliablility and speed.
+
+Open source Spark and Databricks' Spark are very similar, and this provides
+a good enough tests for the SQL parts of a backend.
```

Comment:
s/a good/good

Reply:
Mention that the Databricks CE is too flakey and not similar enough to production to use in tests locally, too?

Reply:
Have rolled this into to my suggested improvement below.